### PR TITLE
Introduce ConditionRelationToggle and use it in QueryCondition

### PR DESCRIPTION
**Pull request by Ladsgroup**
Currently, since the sparql part not merged yet, this won't work but it fully works with the state.

Bug: T272981

### CODE REVIEW

File: tests/unit/components/QueryBuilder.spec.ts [hunk 1]
```diff
@@ -58,4 +60,36 @@ describe( 'QueryBuilder.vue', () => {
 		const wrapper = shallowMount( QueryBuilder, { store, localVue } );
 		expect( wrapper.find( '.querybuilder__condition-placeholder' ).exists() ).toBeFalsy();
 	} );
+
+	it( 'shows the "or" in toggle when there is two conditions', () => {
```
Comment (guergana) [inline tests/unit/components/QueryBuilder.spec.ts]: a wording thingy: 
shows the "or" in toggle when there ARE two conditions

**Review by guergana — CHANGES_REQUESTED** [review]

File: tests/unit/components/QueryBuilder.spec.ts (hunk 1 above)
Comment (Ladsgroup) [reply tests/unit/components/QueryBuilder.spec.ts]: oh right. Done.

File: src/components/QueryBuilder.vue [hunk 2]
```diff
@@ -168,6 +197,30 @@ $largeViewportWidth: 90em; //~1438px
...
+	border-inline: $border-width-thin $border-style-base $border-color-base-default;
+	background-color: $color-base-80;
+}
+
+.querybuilder__condition-relation-toggle-or {
+	padding-inline-start: $dimension-layout-xsmall;
+}
+
+.querybuilder__condition-relation-toggle {
+	padding-block: $dimension-layout-xsmall;
```
Comment (guergana) [inline src/components/QueryBuilder.vue:221]: I am testing it and I see the and has the spacing and the or is on the left. I think the logic is inverted. 

image

**Review by guergana — CHANGES_REQUESTED** [review]

File: src/components/QueryBuilder.vue (hunk 2 above)
Comment (Ladsgroup) [reply src/components/QueryBuilder.vue:221]: Ugh. It was correct but once I was done, I did some refactoring that broke it. It should be fixed now.

**Review by guergana — APPROVED** [review]
Looks really cool

**Pull request merged**